### PR TITLE
Add C# Example Code For Creational Design Patterns

### PR DESCRIPTION
**Pull request by RezaMansouri70**
The sample code of C# programming languages has been added and in the next pull request ,  C# sample code for structural and behavioral design patterns will be added.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -409,6 +462,58 @@ marketingManager.takeInterview();
...
+    public void TakeInterview()
+    {
+        var interviewer = this.MakeInterviewer();
+        interviewer.AskQuestions();
+    }
+}
+
+----------------------------
+
+var devManager = new DevelopmentManager();
```
Comment (Rezakazemi890) [inline README.md:516]: hi, Thanks for add C# samples. 
I think Implementation Of DevelopmentManager is missing.

File: README.md [hunk 2]
```diff
@@ -409,6 +462,58 @@ marketingManager.takeInterview();
...
+devManager.TakeInterview(); //Output : Asking about design patterns!
+
+var marketingManager = new MarketingManager();
```
Comment (Rezakazemi890) [inline README.md:519]: hi, Thanks for add C# samples. 
I think Implementation Of MarketingManager is missing.

**Review by Rezakazemi890 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (RezaMansouri70) [reply README.md:516]: hi , thanks for noticing , DevelopmentManager  Class has been added.

File: README.md (hunk 2 above)
Comment (RezaMansouri70) [reply README.md:519]: hi , thanks for noticing , MarketingManager Class has been added.

**Comment by 3lf** [comment]
Dear Reza,

Thanks for contributing.

I'm waiting for you to finish your work

**Comment by 3lf** [comment]
Dear Reza,
Thanks for contributing.
It would be great if you could finish your work soon.
Thank you.

**Comment by RezaMansouri70** [comment]
Sure :)

**Comment by RezaMansouri70** [comment]
All C# Sample Code Are Added .

**Comment by RezaMansouri70** [comment]
Done

**Pull request merged**